src/yafowil/plone/connectors.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-from collections import MutableMapping
+from collections.abc import MutableMapping
```

Comment:
can we merge this PR into https://github.com/bluedynamics/yafowil.plone/pull/29?